backend/src/main/java/gov/cdc/usds/simplereport/api/testresult/TestResultMutationResolver.java

```diff
@@ -15,13 +16,23 @@
 @Slf4j
 public class TestResultMutationResolver {
   private final TestEventRepository testEventRepository;
+
+  @Qualifier("csvQueueReportingService")
   private final TestEventReportingService testEventReportingService;
 
+  @Qualifier("fhirQueueReportingService")
+  private final TestEventReportingService fhirReportingService;
+
   @MutationMapping
   public boolean resendToReportStream(@Argument List<UUID> testEventIds) {
     testEventRepository
         .findAllByInternalIdIn(testEventIds)
-        .forEach(testEventReportingService::report);
+        .forEach(
+            testEvent -> {
+              testEventReportingService.report(testEvent);
+              fhirReportingService.report(testEvent);
```

Comment:
I think we need some additional filtering here as well - not all tests should be(re)sent to the FHIR queue/universal pipeline, only those including flu results.

As an aside, it doesn't technically affect anything down the line to include the FHIR export everywhere since ReportStream will filter all covid tests out of the universal pipeline anyways. However, it is a performance hit to unnecessarily convert everything to FHIR and put it on the Azure queue if we don't really need it there. The performance is less important here since this is an internal-only mutation, but I think I'd still prefer not to export unless it's necessary.

Reply:
I think that's the eventual end goal, but we probably have a loooong way to go before the Universal Pipeline is ready to take over sending covid-19 test data. (I'm not certain of the details, but it might require them to sort of re-connect with the various STLT receivers, not to mention port over all the covid-specific logic to the UP.) But I agree with you that when we get to that point we can just remove all the CSV reporting logic entirely.